### DISCUSSION TITLE
Forced combinations don't work with nesting folders

### DISCUSSION THREAD

**Discussion by grvnft** [nftchef/art-engine#239] [Help + Q&A]
Hey everyone!
I have some troubles with forced combinations. First it worked fine, then I renamed pngs (files and in config.js) and added nestings folders and it stopped working. Is nesting folders and forced combinations work together?
Pfa examples of my config and folders.
<img width="546" alt="scr1" src="https://user-images.githubusercontent.com/107717343/187481869-ae522105-0c28-4349-b26e-45f66ecdd0ea.png">
<img width="268" alt="scr2" src="https://user-images.githubusercontent.com/107717343/187482220-214211a5-2285-4ae3-87ee-aa78dbb47e03.png">

**Comment by ranovr** [comment]
Having the same issue

**Reply by grvnft** [reply]
up
